qemu/tests/block_aio_io_uring.py

```diff
@@ -0,0 +1,77 @@
+"""Blockdev aio=io_uring basic test"""
+
+
+from provider.block_devices_plug import BlockDevicesPlug
+from virttest import utils_disk, utils_misc
+from virttest.utils_misc import get_linux_drive_path
+from virttest.utils_windows.drive import get_disk_props_by_serial_number
+
+
+def run(test, params, env):
+    """
+    VDPA blk vhost vdpa test. It tests multiple scenarios:
```

Comment:
@qingwangrh  This doc seems not match the case itself, not related to the VDPA ,please update it. Also for the  'steps' below.

Reply:
updated